gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=13.0.5
+version=13.1.0
```

Comment:
Setting this as a minor change as it shouldn't break any existing integration, but given the feature itself I'm ok if we want to release a new major version

Reply:
Minor looks good to me!